### ISSUE TITLE
Determine whether a CELO/asset1 trading limit is helpful

### ISSUE THREAD

**Issue by tobikuhlmann** [mento-protocol/mento-core#173]
**Description**
In [trading limits configs](https://github.com/mento-protocol/mento-core/issues/119) there are limits set for asset0, the first token in a pool. In most cases this is the stable token of the pair. 

Does it add further protection against exploits to also set a trading limit for the second token in a pool? This would be mostly for CELO. There is an ongoing discussion between Nadiem, Roman, Tobi.  

A possible exploit scenario: If the price of CELO goes down in a way that the circuit breaker doesn't trigger, all reserve CELO would be exposed, as one stable token becomes worth a lot of CELO and the current trading limit only measure stable token. A bit protection comes from the bucket sizes, though. The question here is whether we want to expose all the reserve CELO for trading, or limit the CELO exposure as we probably want most of the trading happening with USDC anyways. 

It's possible for example to expose half or quarter of what's in the reserve in CELO. Of the ~120 million CELO tokens only allow 60 or less for trading. Also as an investment risk management strategy for the reserve. 

For a McMento upgrade, a new function could be helpful where the trading limit of CELO is not calculated based on #tokens or current oracle price, but on a time weighted average.

**Acceptance Criteria**
- [ ] Determine whether to enable a trading limit for the second tokens in pools

**Comment by tobikuhlmann** [comment]
goal: Limit reserve CELO trading exposure

**Comment by tobikuhlmann** [comment]
Closing this issue after I got Nadiem's opinion on it. We agreed that this is not urgent. For a future code upgrade, risk could likely be reduced by introducing dynamic parameters: dynamic spread based on reserve rebalancing cost, dynamic minting, meaning minting with collateral assets other than stables becomes only possible if collaterization ratio is above 1, and such. But this is an innovation topic.
